### ISSUE TITLE
Allow to specify supported file extensions in static file handler

### ISSUE THREAD

**Issue by asvetlov** [aio-libs/aiohttp#773]
Like Tornado:

```
application.append((r"/(.*[.](?=ico$|js$|css$).*$)", tornado.web.StaticFileHandler, {"path": "."}))
```

**Comment by asvetlov** [comment]
Depends on #839

**Comment by asvetlov** [comment]
Fixed by #901

**Comment by asvetlov** [comment]
#901 doesn't add tornado-style static file handler but introduces a `FileSender` class which allows to write file sending code easy.
